generated/ws-api/websocket-trade-api.md

```diff
@@ -24,6 +24,100 @@ and positions, and then can begin submitting
 Application-level heartbeats are expected every 30 seconds. If more than one
 interval is missed, the order service will disconnect the websocket.
 
+### Price, Quantity, and Lots
```

Comment:
I intentionally moved this section was to `trade-api.md` because a lot of people reading the REST documentation kept asking about these concepts and it's not clear that REST users would need to read the WS docs.

Not seeing this on `staging` trade.proto e.g. - could you check what you're genning against?

Reply:
Resolved, fixed script to point at correct repo